src/main/java/uk/co/ramp/covid/simulation/place/CareHome.java

```diff
@@ -14,7 +14,7 @@
     public CareHome(Size s) {
         super(s);
 
-        transConstant = PopulationParameters.get().buildingProperties.careHomeTransmissionConstant;
+        transAdjustment = PopulationParameters.get().buildingProperties.careHomeTransmissionConstant;
```

Comment:
transAdjustment seems mis-named to me, or is it careHomeTransmissionConstant that is misnamed?

Reply:
Discussed this wiht Blair. Adjusted to transAdjustment to make it consistent wiht other Place() Classes. Will add an issue to sort htes eout

Reply:
Agree we should keep this consistent now and fix the names in a future (relatively high priority) issue